### ISSUE TITLE
default expansion level

### ISSUE THREAD

**Issue by ksmth** [Lapple/react-json-inspector#4]
currently the inspector only expands the first level. it'd be nice to specify the initial depth that is unfolded.

**Comment by Lapple** [comment]
Added somewhat similar in 3d7e5f6. Now you can pass `isExpanded` property to `Inspector` component, for example, to define initial expansion depth, try using:

``` jsx
function getKeypathDepth(keypath) {
    var dots = keypath.match(/\./g);
    return dots ? dots.length : 0;
}
function isExpanded(keypath) {
    return getKeypathDepth(keypath) < 3;
}
// ...
<Inspector data={ data } isExpanded={ isExpanded } />
```

Edit: Released as `react-json-inspector@4.3.0`.

**Comment by ksmth** [comment]
Perfect!
